### PR TITLE
[ticket/9211] Use different icon for link subforum in forum legend

### PR DESCRIPTION
**Pull request by marc1706**
Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-9211

PHPBB3-9211

### CODE REVIEW

File: phpBB/includes/functions_display.php [hunk 1]
```diff
@@ -486,10 +490,21 @@ function display_forums($root_data = '', $display_moderators = true, $return_mod
 
 				if ($subforum_row['display'] && $subforum_row['name'])
 				{
+					if ($subforum_row['type'] == FORUM_LINK && ($subforum_row['flags'] & FORUM_FLAG_LINK_TRACK) == false
+						&& empty($subforum_row['password']) && $auth->acl_get('f_read', $subforum_id))
+					{
+						$link = $subforum_row['link'];
```
Comment (Nicofuma) [inline phpBB/includes/functions_display.php]: Why these changes? It seems to be unrelated

Comment (marc1706) [inline phpBB/includes/functions_display.php]: The idea was to directly show the target URL but we could of course just use the previous syntax and let view forum do the redirect

**Comment by marc1706** [comment]
Updated

File: phpBB/includes/functions_display.php (hunk 1 above)
Comment (Nicofuma) [inline phpBB/includes/functions_display.php]: Well it was just to know ;)
But it seems I missed your answer
